utils/data-migration/helpers/database.js

```diff
@@ -0,0 +1,191 @@
+  if (!connection) await connect();
+  let rows;
+
+  if (raw) {
+    rows = await connection.collection(name);
+  } else {
+    rows = filter ? await connection.collection(name).find(filter).toArray() : await connection.collection(name).find({}).toArray();
+  }
+
+  return new Promise((resolve, reject) => {
+    if (rows) {
+      resolve(rows);
+    } else {
+      reject(new Error(`Unable to get collection ${name}`));
+    }
+  });
+};
+
+/**
+ * Ascertain UKEF deal ID property path
+ * as per deal type.
+ * @param {String} dealType Deal Type
+ * @returns {String} UKEF Deal ID property path
+ */
+const ukefDealIdPath = (dealType) => {
+  switch (dealType) {
+    case 'GEF':
+      return 'dealSnapshot.ukefDealId';
+    default:
+      return 'dealSnapshot.details.ukefDealId';
+  }
+};
+
+/**
+ * Portal - Updates collection property(ies)
+ * @param {String} ukefDealId UKEF Deal ID
+ * @param {Object} updates Properties to update
+ * @returns {Promise} Resolved as `true` when updated successfully, otherwise Reject error.
+ */
+const portalDealUpdate = async (id, updates) => {
+  try {
+    if (!connection) await connect();
+    const response = await connection.collection(CONSTANTS.DATABASE.TABLES.DEAL).updateOne(
+      { _id: ObjectId(id) },
+      {
+        $set: {
+          ...updates,
+        },
+      },
+    );
+
+    return (response.acknowledged)
+      ? Promise.resolve(true)
+      : Promise.reject(response);
+  } catch (e) {
+    console.error(`Portal deal ${id} update error: `, { e });
+    return Promise.reject(new Error(false));
+  }
+};
+
+/**
+ * Portal facilities - Updates portal facility collection
+ * @param {String} facilityId facility ID
+ * @param {Object} updates Properties to update
+ * @returns {Promise} Resolved as `true` when updated successfully, otherwise Reject error.
+ */
+const portalFacilityUpdate = async (id, updates) => {
+  try {
+    if (!connection) await connect();
+    const response = await connection.collection(CONSTANTS.DATABASE.TABLES.FACILITIES).updateOne(
+      { _id: ObjectId(id) },
+      {
+        $set: {
+          ...updates,
+        },
+      },
+    );
+
+    return (response.acknowledged)
+      ? Promise.resolve(true)
+      : Promise.reject(response);
+  } catch (e) {
+    console.error(`Portal deal ${id} update error: `, { e });
+    return Promise.reject(new Error(false));
+  }
+};
+
+/**
+ * TFM - Updates collection property(ies)
+ * @param {Object} updates Properties to update
+ * @returns {Promise} Resolved as `true` when updated successfully, otherwise Reject error.
+ */
+const tfmDealUpdate = async (updatedDeal) => {
+  try {
+    const { _id } = updatedDeal;
+    delete updatedDeal._id;
+
+    if (!connection) await connect();
+
+    const response = await connection.collection(CONSTANTS.DATABASE.TABLES.TFM_DEAL).updateOne(
+      { _id: { $eq: ObjectId(String(_id)) } },
+      { $set: updatedDeal },
+      { returnOriginal: false },
```

Comment:
can we use here `{ returnDocument: 'after', returnNewDocument: true }`?
`returnOriginal` is deprecated

Reply:
Have added.